docs/01-getting-started.md

```diff
@@ -159,7 +155,7 @@ The VPC _must_ have an S3 Gateway endpoint associated with it otherwise the `cdk
 While connected to your EC2 instance run:
 
 ```bash
-sleeper deployment test/deployAll/deployTest.sh ${ID} ${VPC} ${SUBNETS}
+sleeper builder sleeper/scripts/test/deployAll/deployTest.sh ${ID} ${VPC} ${SUBNETS}
```

Comment:
This requires the system to have been built, but there's no step in the instructions to do that.